modules/govuk/manifests/app/config.pp

```diff
@@ -148,18 +142,6 @@
         value   => $sentry_dsn;
     }
 
-    if $cache_api_calls {
-      $ensure_disable_cache_var = 'absent'
-    } else {
-      $ensure_disable_cache_var = 'present'
-    }
-
-    govuk::app::envvar { "${title}-GDS_API_DISABLE_CACHE":
-      ensure  => $ensure_disable_cache_var,
-      varname => 'GDS_API_DISABLE_CACHE',
-      value   => "${!cache_api_calls_final}",
```

Comment:
I'm a little confused, do you know where the code went that was setting this variable from hiera?

Reply:
Yeah, it went away when we deployed to production: https://github.com/alphagov/govuk-puppet/pull/8036